src/components/list-section/index.tsx

```diff
@@ -3,19 +3,17 @@ import Image from 'next/image';
 
 const services = [
   {
-    title: `AI Implementation`,
-    description: `Unlock the full potential of artificial intelligence in your organization
-     with our expert AI implementation services. 
-     From initial assessment to seamless integration,
-     we ensure that AI technologies are effectively embedded into your business processes,
-     enhancing decision-making, efficiency, and innovation.`,
+    title: `Business Plan AI Helper`,
+    description: `Jumpstart your venture with a solid foundation using our Business Plan Generator.
+    Transform your vision into a detailed, actionable business plan that outlines your pathway to success.
+    Whether you're seeking funding or simply setting strategic goals, our tool guides you through the process,
+    ensuring every critical aspect of your startup is covered.`,
   },
   {
-    title: `Business Scaling`,
-    description: `Scale your business with confidence and precision.
-     Our business scaling services provide you with the strategies and frameworks needed to manage growth effectively,
-     maintain quality, and expand your operations.
-     We help you navigate the challenges of scaling to reach new markets and increase your market share.`,
+    title: `OKRs AI Assistance`,
+    description: `Set objectives and key results that align with your vision and drive your team forward.
+    Our OKRs Assistance simplifies the process of establishing, tracking, and achieving your key business outcomes.
+    Foster a culture of accountability and continuous improvement as you scale.`,
```

Comment:
Our HR tools streamline your people management processes, helping you build a strong, motivated team. The Employee Evaluation feature enables fair, structured assessments, fostering growth and accountability. With the HR Management Plan, you can align your people strategies with business goals, ensuring smooth operations and a supportive workplace culture. The Recruitment tool simplifies talent acquisition, allowing you to find and onboard the right people efficiently. Finally, our Retention Plan provides actionable insights and strategies to keep your top talent engaged and committed, reducing turnover and strengthening team stability